Gemfile

```diff
@@ -5,11 +5,11 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby File.read(".ruby-version")
 
-gem "rails", "~> 7.0.4", ">= 7.0.4.1"
+gem "rails", "~> 7.0.8", ">= 7.0.8.1"
 
 gem "sqlite3", "~> 1.4" # NOTE: Default rails minimum version requirement
 
-gem "puma", "~> 5.0" # NOTE: Default rails minimum version requirement
```

Comment:
You don't actually need this change. If you upgrade puma in the Gemfile.lock, it'll use the higher version.

The "normal" path to do this kind of maintenance-level upgrade would probably just do `bundle update --all`, and let it do its job.